bzt/modules/jmeter.py

```diff
@@ -92,10 +94,14 @@ def prepare(self):
         self.distributed_servers = self.execution.get('distributed', self.distributed_servers)
         scenario = self.get_scenario()
         self.resource_files()
+        system_props = self.settings.get("system-properties")
         if Scenario.SCRIPT in scenario:
             self.original_jmx = self.__get_script()
             self.engine.existing_artifact(self.original_jmx)
         elif "requests" in scenario:
+            if system_props:
+                if not system_props.get("sun.net.inetaddr.ttl"):
```

Comment:
Why do we set proprty here? If no prop has been set - then we should not set it.